metrics/metrics.go

```diff
@@ -0,0 +1,66 @@
+package metrics
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var (
+	MOutgoingMsgs = stats.Int64("pubsub/outgoing_msg_size", "Outgoing message size", stats.UnitBytes)
+	MIncomingMsgs = stats.Int64("pubsub/incoming_msg_size", "Incoming message size", stats.UnitBytes)
+	MTopics       = stats.Int64("pubsub/topics", "Number of topics currently subscribed", stats.UnitDimensionless)
+	MPeers        = stats.Int64("pubsub/peers", "Number of pubsub peers", stats.UnitDimensionless)
+
+	OutgoingMsgCountView = &view.View{
```

Comment:
I am not sure exactly what would make sense, but I would like to at least have the option to differentiate these views by `peer_id` see [here](https://github.com/libp2p/go-libp2p-kad-dht/pull/317/files#diff-a4f6a21f1d4b814bb22ae5c16546906dR18) and [here](https://github.com/libp2p/go-libp2p-kad-dht/pull/317/files#diff-a4f6a21f1d4b814bb22ae5c16546906dR48).